contracts/external/interfaces/gmx/GmxDeposit.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.9;
+
+// @title Deposit
+// @dev Struct for deposits
+library Deposit {
```

Comment:
Prefix the name with `Gmx`. 2 new-lines between contract/library name & pragma/imports